unit/java_bytecode/java_bytecode_parse_generics/class_bound.java

```diff
@@ -0,0 +1,8 @@
+class C{
+}
+interface I{
+}
+class interface_bound<T extends I> {
+}
+class class_bound<T extends C> {
```

Comment:
Could you add a test that has both a class *and* interface bound:

```java
class both_bound<T extends C & I> {
}
```

Reply:
we currently don't support this, it is parsed, but falls back to the descriptor, same for
```
interface I {
}
interface J {
}
class both_bound<T extends J & I> {
}
```

Reply:
You can add a TODO note to the test file for now and refer to TG-1286.